common/utils/src/main/resources/error/error-classes.json

```diff
@@ -492,15 +492,15 @@
     },
     "sqlState" : "54000"
   },
-  "COLUMN_ALIASES_IS_NOT_ALLOWED" : {
+  "COLUMN_ALIASES_NOT_ALLOWED" : {
```

Comment:
I considered naming this `COLUMN_ALIASES_ARE_NOT_ALLOWED`, but since these are slugs they don't really need proper grammar.

This change at least corrects the incorrect combination of the plural "aliases" with the singular "is".